similarity-data-impl/src/main/java/org/phenotips/data/similarity/script/ExomiserViewScriptService.java

```diff
@@ -51,6 +54,16 @@
 @Singleton
 public class ExomiserViewScriptService implements ScriptService
 {
+    private static final int MAXIMUM_NUMBER_OF_GENES = 10;
```

Comment:
The real maximum is 500, this is just a restricted maximum for unprivileged users. Use a better name.